app/controllers/autotune/projects_controller.rb

```diff
@@ -168,17 +168,12 @@ def build_data
 
       @build_data = request.method == 'POST' ? request.POST : @project.data
 
-      # Bust google doc cache if we are forcing an update
-      if @build_data['google_doc_url'].present? && request.GET[:force_update]
-        cache_key = "googledoc#{GoogleDocs.key_from_url(@build_data['google_doc_url'])}"
-        Rails.cache.delete(cache_key)
-      end
-
       # Get the deployer object
       deployer = @project.deployer(:preview, :user => current_user)
 
       # Run the before build deployer hook
-      deployer.before_build(@build_data, {})
+      # set an env var to bust google doc cache if we are forcing an update
+      deployer.before_build(@build_data, 'FORCE_UPDATE' => request.GET[:force_update].nil?)
```

Comment:
Thoughts on making this an env var? I could move it to an optional parameter on build_data. But then all deployers would need to be updated. 
i didn't want to add it to build_data. That would open up potential for a client to pass in this and cause caching inefficiencies or we'd need code to remove this from project data